PROJECT

```diff
@@ -0,0 +1,7 @@
+version: "2"
+domain: bootstrap.k8s.io
```

Comment:
This should be something under `sigs.k8s.io` and should likely be more easily traceable back to the cluster-api subproject.

Reply:
bootstrap.clusterapi.sigs.k8s.io ?

Reply:
I think @vincepri used `cluster.sigs.k8s.io`, so maybe `bootstrap.cluster.sigs.k8s.io`?

Reply:
`bootstrap.cluster.sigs.k8s.io`

Reply:
works for me